README.md

```diff
@@ -129,7 +129,7 @@ to resources.
 
     <!--This section needs a description.-->
 
-    - [The Sound of the Romanian Language](https://youtube.com/watch?v=3rBy6c7UM08)
+    - [The Sound of the Romanian Language](https://youtube.com/watch?v=GF4zhbIJmA8)
```

Comment:
It's peculiar that the video is just straight up unavailable, but in a case like this one where it doesn't seem there is a reupload, we can just remove it. We're not replacing it with an Istro-Romanian variant. 😂

Reply:
To save some time, I just went ahead and removed it so we can merge it.